routers/actionsRouter.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../data/helpers/actionHelpers.js");
+
+router.post("/", async (req, res) => {
+  try {
+    const action = req.body;
+    if (!action.action_description || !action.project_id) {
```

Comment:
Typically it's favored to have positive conditions; so, with your `if... else`, you could switch it around to achieve this:
```javascript
if (action.action_description && action.project_id) {
  const posted = await db.insert(action);
  res.status(201).json(posted);
} else {
  res
    .status(400)
    .json({ error: "Please provide an action name and valid project id" });
}
```